package.json

```diff
@@ -193,29 +194,29 @@
     "split2": "^4.2.0",
     "standard": "^17.1.0",
     "tap": "^16.3.9",
-    "tsd": "^0.29.0",
-    "typescript": "^5.2.2",
-    "undici": "^5.26.0",
+    "tsd": "^0.31.0",
+    "typescript": "^5.4.5",
+    "undici": "^6.13.0",
     "vary": "^1.1.2",
-    "yup": "^1.3.2"
+    "yup": "^1.4.0"
   },
   "dependencies": {
     "@fastify/ajv-compiler": "^3.5.0",
-    "@fastify/error": "^3.4.0",
+    "@fastify/error": "^3.4.1",
     "@fastify/fast-json-stringify-compiler": "^4.3.0",
     "abstract-logging": "^2.0.1",
     "avvio": "^8.3.0",
     "fast-content-type-parse": "^1.1.0",
```

Comment:
This is no longer used and should be removed after https://github.com/fastify/fastify/pull/4450

Reply:
Cleaning up of dependencies can happen subsequent to this PR. This PR is about updating `main` for v5 and is already big enough.

Reply:
Sure, was just documenting for future reference